src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -68,6 +68,7 @@
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
     <PackageReference Include="Microsoft.Azure.Storage.File" Version="11.1.7" />
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.26" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.10" />
```

Comment:
This appears to be the only source code behavior change now. What exactly is in this version of the HTTP extension that fixes this behavior?

Reply:
Take a look at the other PR I sent for that extension